### PR TITLE
fix(amazonq): file opening behavior on accepted changes

### PR DESCRIPTION
**Pull request by kelvin-klchu**
## Problem
- when you accept an individual code change, it will bring up the file window
- when you update the accepted file, both left and right of the window reflects the changes you made immediately
- it should only show the changes made on the left, while the right stays the same as the original suggested code change

## Solution
- updated the way vscode.diff is called when opening up the file such that it is the actual file on the left, and the suggested code change stays on the right

---

License: I confirm that my contribution is made under the terms of the Apache 2.0 license.

### CODE REVIEW

**Review by jpinkney-aws — APPROVED** [review]

**Comment by jpinkney-aws** [comment]
I'll wait for approval from your team before merging

**Review by ctidd — APPROVED** [review]

**Pull request merged**
